src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala

```diff
@@ -587,11 +596,14 @@ object SbtBundle extends AutoPlugin {
   private def formatServiceName(serviceName: String): String =
     s"""        service-name  = "$serviceName""""
 
-  private def formatEndpoints(endpoints: Map[String, Endpoint]): String = {
+  private def formatEndpoints(bundleConfVersion: BundleConfVersions.Value, endpoints: Map[String, Endpoint]): String = {
     val formatted =
       for {
         (label, Endpoint(bindProtocol, bindPort, services, serviceName, acls)) <- endpoints
       } yield {
+        if (acls.exists(_.nonEmpty) && bundleConfVersion != BundleConfVersions.V_1_2_0)
```

Comment:
We should also check that the `services` are empty if version 1.2 or above is used. Otherwise we would get a broken `bundle.conf` if `services` and `acls` are set.